src/js/common/schemaform/SaveInProgressIntro.jsx

```diff
@@ -59,7 +59,7 @@ export default class SaveInProgressIntro extends React.Component {
     if (profile.loading && !this.props.resumeOnly) {
       return (
         <div>
-          <LoadingIndicator message="We’re checking to see if you have a saved version of this application"/>
+          <LoadingIndicator message="We’re checking to see if you have a saved version of this application."/>
```

Comment:
Since the rest are gerund phrases with ellipses maybe:

`Checking to see if you have a saved version of this application...` here?